.travis.yml

```diff
@@ -19,10 +19,6 @@ addons:
     - gettext
 
 before_install:
-- openssl aes-256-cbc -K $encrypted_ead445d7a1e2_key -iv $encrypted_ead445d7a1e2_iv
-  -in gcp-credentials.json.enc -out gcp-credentials.json -d
-- openssl aes-256-cbc -K $encrypted_ead445d7a1e2_key -iv $encrypted_ead445d7a1e2_iv
-  -in application_secrets.json.enc -out application_secrets.json -d
```

Comment:
Since you're getting rid of both of these, I think you can remove the `secure` lines under `env.global`.

Reply:
Fixed.